common/src/WooCommerce.php

```diff
@@ -60,8 +61,11 @@ public function autoSyncNotice() {
     }
 
     public function set_order_language($order_id) {
-        if (!get_post_meta($order_id, 'wpml_language') && defined('ICL_LANGUAGE_CODE')) {
-            update_post_meta($order_id, 'wpml_language', ICL_LANGUAGE_CODE);
+        /** @var WC_Order $order */
+        $order = wc_get_order($order_id);
+        if (!$order->get_meta('wpml_language') && defined('ICL_LANGUAGE_CODE')) {
+            $order->update_meta_data('wpml_language', ICL_LANGUAGE_CODE);
+            $order->save();
```

Comment:
I had a look at the WooCommerce source code and it looks like `save_meta_data()` does a bit less than `save()` and prevents some caches from being cleared. So I suggest we use that instead.

```suggestion
            $order->save_meta_data();
```

Reply:
This is marked as resolved, but I still see `save()` instead of `save_meta_data()`?